.github/workflows/p2p-get-latest-image.yaml

```diff
@@ -64,8 +64,9 @@ jobs:
           yq -p=json -C <<EOT
           ${{ toJSON(env) }}
           EOT
+
       - name: Checkout
```

Comment:
we can probably just remove the checkout for this workflow?

Reply:
yeah true